plugins/Diagnostics/lang/en.json

```diff
@@ -9,6 +9,8 @@
         "HideUnchanged": "If you want to see only changed values you can %1$shide all unchanged values%2$s.",
         "Sections": "Sections",
         "DatabaseReaderConnection": "Database Reader Connection",
+        "DatabaseUtf8mb4CharsetRecommended": "Your database doesn't support utf8mb4 charset. This is required to be able to store four byte UTF8 characters. Unless utf8mb4 is available those characters will be replaced.",
```

Comment:
be good to clarify the `will be replaced` so it's clear, maybe something like `will be removed from the Tracking API values. For example special characters such as Emojis, x, y, z will be removed from your Page URLs, Page titles, custom dimension values, etc.`

Reply:
Actually those chars are currently replaced with `�`. updated the text and added some examples